web_src/js/components/DiffFileTreeItem.vue

```diff
@@ -59,10 +59,13 @@ a, a:hover {
 
 .sub-items {
   padding-left: 9px;
+  margin-left: 9px;
+  border-left: 1px solid #808080;
```

Comment:
```suggestion
  border-left: 1px solid var(--color-secondary-dark-5);
```

We should not use bare colors in CSS.

Reply:
> I'm surprised this had passed lint.

After doing some research, I saw change `color-no-hex` to `true` and exclude theme**.css from the rule[(adding a .sytlelintignore or add comments)](https://stylelint.io/user-guide/ignore-code) should be a possible fix. 
https://github.com/go-gitea/gitea/blob/c37f95fc56fa3fc8f51610af6915ac39839d7d19/.stylelintrc.yaml#L29-L34
Want me to send a PR or do you have a better solution?

Reply:
The ideal rule for this is https://github.com/AndyOGo/stylelint-declaration-strict-value, but I see it only works on rules like `border-color`, not shorthands like `border-left`. If you find a rule config for it that works, go ahead. It might reveal more violations.

Reply:
It seems we would need to enable experimental shorthand support:

https://github.com/AndyOGo/stylelint-declaration-strict-value#css-shorthand-syntax